src/wasm2asm.h

```diff
@@ -578,6 +583,49 @@ void Wasm2AsmBuilder::addExports(Ref ast, Module* wasm) {
   ast->push_back(ValueBuilder::makeStatement(ValueBuilder::makeReturn(exports)));
 }
 
+void Wasm2AsmBuilder::addGlobal(Ref ast, Global* global) {
+  assert(global->init);
```

Comment:
no need for this, it would be invalid IR if it didn't exist (which should be caught in general IR verification)